includes/wikia/services/ArticleService.class.php

```diff
@@ -155,7 +155,7 @@ function() use ( $service, $fname ) {
 					wfProfileIn( $fname . '::CacheMiss' );
 
 					$content = '';
-					if ( !empty( $this->wg->SolrMaster ) ) {
+					if ( !$this->wg->DevelEnvironment && !empty( $this->wg->SolrMaster ) ) {
```

Comment:
Added this extra logic so that we don't query Solr in the dev environment and instead fall back to the code on line 162 below.  We have no dev Solr instance so this code will query production.  Its very likely that production will already have an article indexed with the ID just created here in dev and will return incorrect content.